src/html/index.html.template

```diff
@@ -53,14 +47,10 @@
           background-color: #111111;
           color: #e1e1e1;
         }
-        #ha-init-skeleton::before {
-          background-color: #1c1c1c;
```

Comment:
This color should be updated, as it is wrong now: https://github.com/home-assistant/frontend/blob/dev/src/resources/styles.ts#L11